src/dataflow/core/lispress.py

```diff
@@ -390,12 +342,20 @@ def _program_to_unsugared_lispress(program: Program) -> Lispress:
         # create a sexp for expression
         idx = expression.id
         op_lispress = op_to_lispress(expression.op)
+        op_type_args_lispress = type_args_to_lispress(expression.type_args)
+        # if there type args, we create a META expression
+        if op_type_args_lispress is not None:
+            op_lispress = [META_CHAR, op_type_args_lispress, op_lispress]
         curr: Sexp
         if isinstance(expression.op, (BuildStructOp, CallLikeOp)):
             curr = [op_lispress]
-            named_args = sorted(get_named_args(expression))  # sort alphabetically
+            named_args = get_named_args(expression)
+            # if all args are named (i.e., not positional), sort them alphabetically
```

Comment:
don't we want to sort the subset of args that are named args no matter what?

Reply:
I'll ask @eaplatanios , who I'm pretty sure wrote this.